.env.example

```diff
@@ -13,9 +13,11 @@ REQUEST_SHEET_DATA_RANGE='Request Private Beta'!A1
 REQUEST_SHEET_HEADER_RANGE='Request Private Beta'!A1:Y1
 
 # Zendesk
-ZENDESK_EMAIL=a-zendesk-account@digital.cabinet-office.gov.uk
+ZENDESK_API_URL: https://govuk.zendesk.com/api/v2
+ZENDESK_USERNAME=a-zendesk-account@digital.cabinet-office.gov.uk
 ZENDESK_API_TOKEN='An API token that works with that email address'
-ZENDESK_TAG='A tag used to route the ticket to the right team'
+ZENDESK_TAG='A tag used to route the ticket to the right queue'
+ZENDESK_GROUP_ID_PUBLIC='A groupd ID used to assign the ticket to the right group'
```

Comment:
This is not piped through in the PaaS script - assuming it's not needed in the test env because ZD won't be enabled. I wonder if there's a test instance of ZD we could hook it up to, though.

Also rename it to `ZENDESK_GROUP_ID` if it makes sense. Sounds a bit weird named as it is now.

Reply:
I've used the name that Auth are using wherever they use the Zendesk API - I can change it but was going for consistency.  Change or keep?

Reply:
You're right that they're not piped through - they're not needed for the stub but I put them as placeholders as they will be needed eventually and we need for it to work without them being needed / if anyone wanted to run it locally.  Also, there's little to no chance of me remembering what we need by the time we do need them 😆

Reply:
Change. We don't need to be consistent down to variable names.